novatel_gps_driver/CMakeLists.txt

```diff
@@ -74,6 +74,11 @@ target_link_libraries(${PROJECT_NAME}
 )
 set_target_properties(${PROJECT_NAME} PROPERTIES COMPILE_FLAGS "-std=c++11")
 
+### Install Launch Files ###
+install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
You only needed to move the add_dependencies line. The install directive should be kept with the other install commands at the bottom of the file